### DISCUSSION TITLE
package error

### DISCUSSION THREAD

**Discussion by Bookreflect** [tw93/Pake#725] [General]
![1717492566847](https://github.com/tw93/Pake/assets/23049821/b51401f8-b696-4e36-a530-1cd755c0a722)

**Comment by kaixiangliuran** [comment]
上述的问题解决了吗

**Comment by liumu927** [comment]
你这应该是没有配置rust和tauri的环境，看下这里 https://tauri.org.cn/v1/guides/getting-started/prerequisites/
